app/views/groups/new.html.erb

```diff
@@ -9,8 +9,11 @@
         <%= render partial: "form_fields", locals: { form: form, group: @new_group } %>
 
         <div class="my-2">
-          <%= form.submit t(:"groups.create.submit"), class: "btn btn-primary mr-1" %>
-          <%= link_to t(:"groups.create.cancel"), groups_path, class: "btn btn-default" %>
+          <%= form.submit t(:"groups.create.submit"),
+                          class: " button button--state-primary button--size-default mr-1" %>
+          <%= link_to t(:"groups.create.cancel"),
+                      groups_path,
+                      class: "button button--state-default button--size-default" %>
```

Comment:
Same as previous comment. One of the buttons is full width.
![image](https://github.com/phac-nml/irida-next/assets/325703/382d238d-a8db-4734-9f8d-395e5e3e25af)

Reply:
 Updated in bd98356